components/authz-service/storage/postgres/rule.go

```diff
@@ -17,6 +18,23 @@ const (
 )
 
 func (p *pg) CreateRule(ctx context.Context, rule *storage.Rule) (*storage.Rule, error) {
+
+	projectsFilter, err := projectsListFromContext(ctx)
+	if err != nil {
+		return nil, err
+	}
+	found := len(projectsFilter) == 0 // empty means all projects allowed
+	if !found {
+		for _, proj := range projectsFilter {
+			if proj == rule.ProjectID {
+				found = true
+			}
+		}
```

Comment:
[nit] This could use [`stringutils. SliceContains`](https://github.com/chef/automate/blob/83e0e071be9f124f2fba5a4915e7109c5d7da5cd/lib/stringutils/slice.go#L7):
```suggestion
		found = stringutils.SliceContains(projectsFilter, rule.ProjectID)
```

Reply:
😎